tests/Cluster.py

```diff
@@ -67,10 +74,13 @@ def __init__(self, walletd=False, localCluster=True, host="localhost", port=8888
         """
         self.accounts={}
         self.clusterID=clusterID
-        if clusterID != 0:
-            port += clusterID * 2000
-            Cluster.__BiosPort = 8788 + clusterID * 2000
-            Cluster.__BootScript = "bios_boot_cluster" + str(clusterID) + ".sh"
+        Utils.clusterID=clusterID
+
+        Cluster.__BiosPort = Cluster.__portBase + clusterID * Cluster.__cluster_stride + Cluster.__port_type_http
+        Cluster.__BootScript = "bios_boot_cluster" + ("%05d" % clusterID) + ".sh"
```

Comment:
I would prefer to see the generated boot script go in the same cluster sub-directory as all the other cluster specific files.